src/axom/quest/MeshTester.cpp

```diff
@@ -93,9 +95,9 @@ void findTriMeshIntersections(detail::UMesh* surface_mesh,
   detail::UniformGrid3 ugrid(resolutions, triBboxes, triIdxs);
 
   // Iterate through triangle indices *idx.
-  // Check against each other triangle with index greater than the index *idx
-  // that also shares a UniformGrid bin.
-  SLIC_INFO("Checking mesh with a total of " << ncells << " cells.");
+  // Check against each other triangle with index greater than the index *idx that also shares a UniformGrid bin.
+  SLIC_INFO(
+    axom::fmt::format(axom::utilities::locale(), "Checking mesh with a total of {:L} cells", ncells));
```

Comment:
Why does locale need to be here?

Reply:
Mostly readability -- it prints out the number of cells w/ comma separators like this:
```
[INFO] Checking mesh with a total of 426,430 cells. 
```